go/storage/mkvs/checkpoint/checkpoint.go

```diff
@@ -125,16 +133,25 @@ type Metadata struct {
 	Chunks  []hash.Hash `json:"chunks"`
 }
 
+// EncodedHash returns the encoded cryptographic hash of the checkpoint metadata.
+func (m *Metadata) EncodedHash() hash.Hash {
+	var hh hash.Hash
+
+	hh.From(m)
```

Comment:
Doesn't need to be now, and we do this all over the place, but at some point we should add `func NewFrom(v interface{}) Hash`.

Reply:
Ah, good idea! I'll do this in a separate PR.

Reply:
Done in #2890.